resources/views/components/input/radio.blade.php

```diff
@@ -1,6 +1,6 @@
-<label class="relative flex w-full cursor-pointer items-center justify-start gap-x-3 rounded border bg-white p-7 text-sm text-ct-primary">
+<label {{ $attributes->only('class')->class("relative flex w-full cursor-pointer items-center justify-start gap-x-3 rounded border bg-white p-2 py-5 sm:p-7 text-sm text-ct-primary") }}>
     {{-- The 0 opacity and 1px width ensures the element is "visible" for the browser so the browser can focus which is useful for validation --}}
-    <input type="radio" {{ $attributes->class('peer opacity-0 border-0 w-px') }}/>
+    <input type="radio" {{ $attributes->except('class')->merge(['class' => 'peer opacity-0 border-0 w-px']) }}/>
```

Comment:
i understand that the radio needs to be invisible but now it still takes up 1px, can you make this absolute with a top-0 and left-0 so it doesn't take any space. Is that also the reason why the element below has a -left-px ? to make up for that space?

Reply:
When you do that the browser validation message will appear at a strange position. Better is to refactor this so the element itself is styled instead of a div next to it.

Reply:
Allright, kind of curious now where it would place the validation. I'll test it after it's merged 👍

Reply:
<img width="559" alt="image" src="https://github.com/rapidez/checkout-theme/assets/1703233/d682938d-e424-46fb-8bdf-1d6c22460d01">